flink-tests/src/test/java/org/apache/flink/test/windowing/sessionwindows/SessionWindowITCase.java

```diff
@@ -114,8 +115,8 @@ private void runTest(
 			windowedStream = windowedStream.allowedLateness(Time.milliseconds(ALLOWED_LATENESS_MS));
 		}
 
-		if (!PURGE_WINDOW_ON_FIRE) {
-			windowedStream = windowedStream.trigger(new NonPurgingEventTimeTriggerWrapper());
+		if (PURGE_WINDOW_ON_FIRE) {
+			windowedStream = windowedStream.trigger(PurgingTrigger.of(EventTimeTrigger.create()));
 		}
```

Comment:
Previously, the correct behavior in the case was still under discussion and not clearly defined. This is why this branch of the test could not yet be properly implemented and should fail as notification that it needs to be adapted once the final behavior is defined.